tests/pds/backup_test.go

```diff
@@ -0,0 +1,85 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	pds "github.com/portworx/pds-api-go-client/pds/v1alpha1"
+	pdsbkp "github.com/portworx/torpedo/drivers/pds/pdsbackup"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+)
+
+const (
+	bkpTargetName = "bkptarget-automation"
+)
+
+var (
+	bkpClient                                                             *pdsbkp.BackupClient
+	awsBkpTargets, azureBkpTargets, s3CompatibleBkpTargets, gcpBkpTargets []*pds.ModelsBackupTarget
+)
+
+var _ = Describe("{ValidateBackupTargetsOnSupportedObjectStores}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("ValidateBackupTargetsOnSupportedObjectStores", "Validate backup targets for all supported object stores.", pdsLabels, 0)
+		bkpClient, err = pdsbkp.InitializePdsBackup()
+		log.FailOnError(err, "Failed to initialize backup for pds.")
+	})
+
+	It("Add all supported Object stores as backup target for data services", func() {
+		stepLog := "Create AWS S3 Backup target."
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			bkpTarget, err := bkpClient.CreateAwsS3BackupCredsAndTarget(tenantID, fmt.Sprintf("%v-aws", bkpTargetName))
+			log.FailOnError(err, "Failed to create AWS backup target.")
+			log.InfoD("AWS S3 target - %v created successfully", bkpTarget.GetName())
+			awsBkpTargets = append(awsBkpTargets, bkpTarget)
+		})
+		stepLog = "Create GCP Backup target."
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			bkpTarget, err := bkpClient.CreateGcpBackupCredsAndTarget(tenantID, fmt.Sprintf("%v-gcp", bkpTargetName))
```

Comment:
how are we validating the backup target is created successfully or not ?
say if its api call we need to we need to assert its response, is that assertion part handled?